### ISSUE TITLE
Trailing comas in function calls not accepted in 1.13 RC

### ISSUE THREAD

**Issue by juanperi** [elixir-lang/elixir#11399]
### Environment

* Elixir & Erlang/OTP versions (elixir --version): 

Erlang/OTP 24 [erts-12.0.4] [source] [64-bit] [smp:16:16] [ds:16:16:10] [async-threads:1] [jit]

Elixir 1.13.0-rc.1 (2765e0f) (compiled with Erlang/OTP 24)

* Operating system: Mac

### Current behavior

The following code does not compile in the RC, but it did compile in 1.12
```elixir
defmodule Test do
  def call(opts) do
    IO.inspect(opts)
  end
end

Test.call(
  a: 1,
  b: 2,
)
```
The error is:
```
$ elixir test.exs
** (SyntaxError) test.exs:10:1: syntax error before: ')'
    |
 10 | )
    | ^
    (elixir 1.13.0-rc.1) lib/code.ex:1183: Code.require_file/2
```

### Expected behavior

Either the code compiles as it did in 1.12, or it is documented as a BC break?
I haven't seen anything related to this change being on purpose. If there was a conversation around it, sorry about the noise!

**Comment by eksperimental** [comment]
I can confirm this. Weird that it allowed a trailing comma in a function call though.

**Comment by juanperi** [comment]
Honestly, after discovering this I went ahead and fixed the 3 cases I had in the codebase. They were not intentional, probably happened after deleting some keyword list parameters in some template files.
It might be that it's considered a bug that is being fixed in 1.13 🤷
